### PR TITLE
added tenantId property and updated sdk

### PR DESCRIPTION
**Pull request by poornima-metron**
1. Added tenantId property in azure_subscription Entity
2. Upgraded SDK to the latest version

Jira Ticket: https://jupiterone.atlassian.net/browse/IA-172

### CODE REVIEW

File: CHANGELOG.md
Comment (suraj-metron) [inline CHANGELOG.md:1]: Did we updated this file or it got automatically updated? I remember J1 team mentioned that this file will be automatically updated. Please correct me If I am wrong

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+
+# v7.8.2 (Fri Aug 16 2024)
```
Comment (suraj-metron) [inline CHANGELOG.md:2]: Did we updated this file or it got automatically updated? I remember J1 team mentioned that this file will be automatically updated. Please correct me If I am wrong

**Review by suraj-metron — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (poornima-metron) [reply CHANGELOG.md:2]: we have updated it , i think the comments was for monorepo PR's

**Comment by Gonzalo-Avalos-Ribas** [comment]
Closing in favor of https://github.com/JupiterOne/graph-azure/pull/671

**Pull request closed without merging**